app/src/main/java/com/mathroule/demoapp/MainActivity.java

```diff
@@ -28,6 +29,13 @@ public void onClick(View view) {
                         .setAction("Action", null).show();
             }
         });
+
+        findViewById (R.id.buton).setOnClickListener(new OnClickListener( ) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/5996d35de4327cd5b1206489937a936390ca85c6/app/src/main/java/com/mathroule/demoapp/MainActivity.java#L33"></span>’)’ is preceded with whitespace.</td>
    </tr>
  </tbody>
</table>